tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2021",
     "module": "node16",
     "moduleResolution": "node16",
-    "declaration": true,
+    "declaration": false,
```

Comment:
we don't need declaration files for this service and they create bloat in the docker container